tests/sentry/utils/performance_issues/test_consecutive_db_detector.py

```diff
@@ -252,3 +252,27 @@ def test_fingerprint_of_autogroups_match(self):
         fingerprint_2 = self.find_problems(event_2)[0].fingerprint
 
         assert fingerprint_1 == fingerprint_2
+
+    def test_detects_consecutive_db_does_not_detect_php(self):
+        span_duration = 50
+        spans = [
+            create_span(
+                "db",
+                span_duration,
+                "SELECT `customer`.`id` FROM `customers` WHERE `customer`.`name` = $1",
+            ),
+            create_span(
+                "db",
+                span_duration,
+                "SELECT `order`.`id` FROM `books_author` WHERE `author`.`type` = $1",
+            ),
+            create_span("db", 900, "SELECT COUNT(*) FROM `products`"),
+        ]
+        spans = [
+            modify_span_start(span, span_duration * spans.index(span)) for span in spans
+        ]  # ensure spans don't overlap
+
+        event = create_event(spans)
```

Comment:
This would be a bit safer if you used a fixture file that's known to return a performance problem. If there happened to be a mistake in this mock that made this event not return a problem (for a reason aside from the SDK) this test would succeed even if the SDK checking logic was faulty

Reply:
Fair! But I did copy a test case from an example above, that does return a performance issue. I can add a fixture in another PR to be extra safe.

Reply:
I could also do two assertions, one with sdk name, and one without!